src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/locationsinsideprison/LocationsInsidePrisonClient.kt

```diff
@@ -19,3 +22,41 @@ class LocationsInsidePrisonClient(private val locationsInsidePrisonApiWebClient:
     .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
     .block()
 }
+
+@Component
+class LocationValidator(private val locationsInsidePrisonClient: LocationsInsidePrisonClient) {
+
+  fun validatePrisonLocation(prisonCode: String, locationKey: String) {
+    validatePrisonLocations(prisonCode, setOf(locationKey))
+  }
```

Comment:
What happens if the person is temporarily out of the prison when the appointment is booked? e.g. he's on day-release or job ROTL? Not sure what prisoner search will tell us where they are?

Reply:
Erm this is validating a location, did you mean that for here???